db/migrate/20240715103415_enable_admin_style_v3_by_default.rb

```diff
@@ -0,0 +1,5 @@
+class EnableAdminStyleV3ByDefault < ActiveRecord::Migration[7.0]
+  def up
+    Flipper.enable(:admin_style_v3)
```

Comment:
I'm not sure why this is not working :disappointed:  Any ideas @dacook?

Reply:
Hi Filipe, only changes to the database schema will affect the test environment, so this won't affect it. 

Instead, we can make it enabled by default using a different method. I'll add a note on the issue to explain (I had to check it to work out how it works mysefl!)

Reply:
However, enabling it by default won't change any existing Dev environments, so this migration would be helpful. But we need to make sure it only applies to dev envs (we don't want to enable on prod just yet!). You could add a condition inside `up` that checks `if Rails.env.development?`